app/Main.hs

```diff
@@ -1,32 +1,41 @@
+{-# LANGUAGE QuasiQuotes #-}
+{-# LANGUAGE TemplateHaskell #-}
```

Comment:
why is this extension needed?

Reply:
`PyF`.

Reply:
hum? we had `PyF` before used in the same way no? I don't see the difference

Reply:
Actually you are right, `QuasiQuotes` seems to be enough.